components/channel/local-env.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Ensure The Script Is Being Sourced
+[[ "${BASH_SOURCE[0]}" == "${0}" ]] && echo -e "\nWARNING: 'Source' this script for the environment setup to apply to the current session!\n"
+
+# Absolute Path Of Directory Containing Script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+# Set The $GOPATH (Up 6 Levels To GO)
+export GOPATH=$(dirname $(dirname $(dirname $(dirname $(dirname $(dirname ${SCRIPT_DIR}))))))
+
```

Comment:
Is this still the correct depth (not sure of the new repo structure)?   Also - maybe should just rely on user to set $GOPATH (this is somewhat legacy).